### ISSUE TITLE
[Crash Report] [1.18.2 Forge] Game keeps crashing once musictriggers.toml is edited in anyway.

### ISSUE THREAD

**Issue by Jirozu** [TheComputerizer/Music-Triggers#20]
Mod load's fine and everything but once I edit the TOML OR use the in-game GUI to add a music trigger it will crash. I remember seeing on the curse forge page that there may still be an issue with mob triggers, so that might be the problem.

TOML File:
```
[withertheme]
	[[withertheme.trigger]]
		name = "mob"
		priority = "15"
		level = "1"
		resource_name = "Wither"
		identifier = "wither"
		detection_range = "150"
```

Crash Report:
[crash-2022-04-15_21.40.36-client.txt](https://github.com/TheComputerizer/Music-Triggers/files/8499071/crash-2022-04-15_21.40.36-client.txt)

**Comment by NekomataGami** [comment]
Same issue here, mod loads just fine with a blank musictriggers.toml, but once I add something, it always crash in-game

What I added:

`[[daytime]]
	[[daytime.trigger]]
		name = "time"
		start = "12"
		end = "18"
`
[latest.log](https://github.com/TheComputerizer/Music-Triggers/files/8501629/latest.log)
[crash-2022-04-17_21.07.07-client.txt](https://github.com/TheComputerizer/Music-Triggers/files/8501630/crash-2022-04-17_21.07.07-client.txt)

**Comment by Huetete** [comment]
This happens to me as well in 1.16

**Comment by NekomataGami** [comment]
I've tested the 1.16.5-5.1 forge version, the mod loads and the music I've configured played just fine. However, the 1.18.2-5.0/5.1 forge version always crashes on launch.

**Comment by TheComputerizer** [comment]
It seems like it is most likely an issue with the mob trigger

**Comment by SeanChengN** [comment]
Same issue here!

**Comment by TheComputerizer** [comment]
This should be fixed in 5.2, or at least easier to debug if it isnt. I will close it for now but if you are still having issues send an updated crash report and I can reopen it
